### PR TITLE
Add a SIGKILL to second ctrl+c

### PR DESCRIPTION
**Pull request by ahopkins**
Hitting `ctrl+c` sends SIGINT to do a graceful shutdown on each worker process. Subsequent `ctrl+c` currently do nothing.

This PR adds functionality so that the second `ctrl+c` will send a SIGKILL to the processes for immediate shutdown.

### CODE REVIEW

**Review by prryplatypus — APPROVED** [review]

File: sanic/worker/manager.py [hunk 1]
```diff
@@ -183,6 +183,11 @@ def kill(self):
         raise ServerKilled
 
     def shutdown_signal(self, signal, frame):
+        if self.terminated:
+            logger.info("Shutdown interrupted. Killing.")
+            with suppress(ServerKilled):
+                self.kill()
+
```
Comment (prryplatypus) [inline sanic/worker/manager.py:190]: Do we want a final `return` at the end of this if statement?

**Comment by Tronic** [comment]
Did you check how this behaves on Windows? IIRC, terminate already works the same as kill, and also KeyboardInterrupt may be received by any of the processes. Also, might consider only handling second Ctrl-C after a slight delay, as it is quite easy to accidentally hit it twice, but this is also not of that much importance (I don't mind if there is no delay required).

**Comment by ahopkins** [comment]
> Did you check how this behaves on Windows? IIRC, terminate already works the same as kill, and also KeyboardInterrupt may be received by any of the processes. Also, might consider only handling second Ctrl-C after a slight delay, as it is quite easy to accidentally hit it twice, but this is also not of that much importance (I don't mind if there is no delay required).

Nope, will check.

File: sanic/worker/manager.py (hunk 1 above)
Comment (ahopkins) [reply sanic/worker/manager.py:190]: Not needed. `self.kill` ends with a `raise`

**Pull request merged**